mirrord/config/src/target/cron_job.rs

```diff
@@ -0,0 +1,51 @@
+use core::fmt;
+use std::fmt::Display;
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use super::{FromSplit, FAIL_PARSE_DEPLOYMENT_OR_POD};
+use crate::config::{self, ConfigError};
+
+#[derive(Serialize, Deserialize, Clone, Eq, PartialEq, Hash, Debug, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct CronJobTarget {
+    pub cron_job: String,
+    pub container: Option<String>,
+}
+
+impl Display for CronJobTarget {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "{}{}",
+            self.container
+                .as_ref()
+                .map(|c| format!("{c}/"))
+                .unwrap_or_default(),
+            self.cron_job.clone()
+        )
+    }
+}
```

Comment:
I don't think this impl is needed, most other target types (excluding `JobTarget`) don't have it :eyes: 
Could you either remove it or change the order of elements (right now the container is displayed before the cron job name). Applies to both `CronJobTarget` and `JobTarget`

Reply:
Removed from both places.